x11/fvwm2/Portfile

```diff
@@ -41,6 +40,12 @@ configure.args      --x-includes=${prefix}/include \
                     --without-rplay-library \
                     --without-stroke-library
 
+# error: incompatible integer to pointer conversion assigning to 'int *' from 'char' [-Wint-conversion]
+if {${os.platform} eq "darwin" && ${os.major} > 22} {
+    configure.cflags-append \
+                    -Wno-error=int-conversion
```

Comment:
Shouldn't you submit a patch upstream where they don't, in fact, attempt to assign to a pointer from a character, which would seem to be an actual bug?

Reply:
They will not accept issues or patches, it is in maintenance mode:

> Please note that FVWM2 is in maintenance mode. This means it won't be receiving any new features. Bug fixes will be to the core of FVWM2 only. Any problems found in modules will be to fix segfaults only.

https://github.com/fvwmorg/fvwm

We can make a port for `fvwm3`, but it does not prevent an update to `fvwm2`.

Reply:
Even if they don't accept a patch, assigning a char to an int * would seem to be a bug. Don't we want code with fewer bugs?

Reply:
Sure. Could you suggest a fix? I will add it then, crediting to you.

Reply:
Not my port; I haven't even read the code. However, the error would seem peculiar. I suggest just looking at it.

Reply:
@barracuda156 does the current version fail with the same error or was this introduced in the update. 

If already present before there is mo real reason not to merge this... Of course, a proper fix would be great but it doesn't look like anyone from the MaPorts side will do that and if upstream isn't either there isn't much one can do.

Merging this while at the same time at least making upstream aware would be great.